configure.ac

```diff
@@ -11,8 +11,8 @@ AC_PREREQ([2.63])
 define([scm_r], esyscmd([sh -c "git rev-parse --short=7 HEAD"]))
```

Comment:
Without git we can't create correct version of UCX. Is that OK to run build on a machine without git?